### PR TITLE
chore: Add Helm chart to PR template

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

Add Helm chart to PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ _Provide a description of what has been changed_
 
 - [ ] Commits are signed with Developer Certificate of Origin (DCO)
 - [ ] Tests have been added
-- [ ] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
+- [ ] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable)*
```
Comment (zroubalik) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
- [ ] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable, ie. when deployment manifests are modified)*
```

I'd add something like this, maybe you can find a better wording

**Comment by zroubalik** [comment]
@tomkerkhove we should have a similar note in charts PR template, to propagate the changes back to the core.

**Comment by tomkerkhove** [comment]
Good point, I'll do that ASAP

**Comment by tomkerkhove** [comment]
> @tomkerkhove we should have a similar note in charts PR template, to propagate the changes back to the core.

Done in https://github.com/kedacore/charts/pull/122

**Review by zroubalik — APPROVED** [review]

**Pull request merged**